bindings/nodejs/__test__/fs.spec.mjs

```diff
@@ -0,0 +1,41 @@
+// Copyright 2022 Datafuse Labs
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+
+import test from 'ava'
+import { Fs } from '../index.js'
+import fs from "node:fs"
+
+const testFolderPath = "./testData"
+const fileContent = "hello world"
+test.before((t) => {
+  fs.mkdir(testFolderPath, () => {})
+})
+
+test.after((t) => {
+  fs.rmdir(testFolderPath, () => {})
+})
+
+test.only("test fs read & write", async (t) => {
+  let builder = new Fs()
+  builder.root = testFolderPath
+  let op = builder.build()
+
+  let o = op.object(t.title)
+  await o.write(new TextEncoder().encode(fileContent))
```

Comment:
`TextEncoder` for test is too verbose, can we use something like `new Uint8Array([21, 31])`?

Reply:
I may pick them up to `before` hook.
It prefers to give a `Buffer` since `napi-rs` will transform the arguments' type.